lib/src/configuration_values.dart

```diff
@@ -0,0 +1,148 @@
+// https://stackoverflow.com/a/73564526
+import 'package:sentry_dart_plugin/src/utils/config-reader/config_reader.dart';
+
+class ConfigurationValues {
+  final String? version;
+  final String? name;
+
+  final bool? uploadDebugSymbols;
+  final bool? uploadSourceMaps;
+  final bool? uploadSources;
+  final String? project;
+  final String? org;
+  final String? authToken;
+  final String? url;
+  final bool? waitForProcessing;
+  final String? logLevel;
+  final String? release;
+  final String? dist;
+  final String? webBuildPath;
+  final String? commits;
+  final bool? ignoreMissing;
+
+  ConfigurationValues({
+    this.version,
+    this.name,
+    this.uploadDebugSymbols,
+    this.uploadSourceMaps,
+    this.uploadSources,
+    this.project,
+    this.org,
+    this.authToken,
+    this.url,
+    this.waitForProcessing,
+    this.logLevel,
+    this.release,
+    this.dist,
+    this.webBuildPath,
+    this.commits,
+    this.ignoreMissing,
+  });
+
+  factory ConfigurationValues.fromArguments(List<String> arguments) {
+    Map<String, String> sentryArguments = {};
+    for (final arg in arguments) {
+      final components = arg.split("=");
```

Comment:
Do we have any values that might contain "=" ? just a thought, this won't work if there are

Reply:
Updated the code and added a test to cover this case.